frontend-react/src/hooks/UseAdminSafeOrganizationName.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from "react";
+
+export enum Organizations {
+    PRIMEADMINS = "PrimeAdmins",
+    IGNORE = "ignore",
+}
+
+/** Helper hook that converts `PrimeAdmins` to `ignore`
+ * @todo Ticket to make PrimeAdmins an RS org {@link https://github.com/CDCgov/prime-reportstream/issues/4140 #4140}
+ * @param orgName {string|undefined} Active membership `parsedName` */
+export const useAdminSafeOrganizationName = (orgName: string | undefined) => {
+    return useMemo(
```

Comment:
This is a very, very pedantic point, but I'm wondering if we need to memoize this since the logic below is basically a simple ternary; I don't know if we'd really benefit much from memoization since a new `orgName` getting passed in would result in a new recomputation (and likely re-render)🤔

Reply:
I also thought the same thing when copy+pasting this from the original file. I didn't want to change anything from the original function from this move initially.

Reply:
Great callout! We were something of a new React team, so we've got a lot of debt in here like "a memo that may not need momization" from bad habits or wonky internet solutions. Definitely want to keep eyes open for chances to fix this, but if the fix becomes cumbersome, reel free to relegate this to a bug/tech-debt item and we can prioritize it in our work that way!

Reply:
Gotcha, sounds good to me!  I do think we can remove the memoization here with no discernible impact, but it's not a huge deal for this case.  (If we remove the memoization, we could also consider making this just a simple function rather than a hook too, but that's out of the scope of this pull request!)

More generally, though, the biggest risks in my mind from over-memoization are 1) data staleness and 2) very, very, very slight performance hit -- the former of which is much more important to deal with from a UX perspective.  BUT again, it isn't a huge deal for this specific case, and we can just keep an eye on this going forward too!